gaminatorium-main/adapter/src/test/java/eu/gaminatorium/game/GameControllerTest.java

```diff
@@ -473,8 +468,7 @@ void deleteGameById() throws Exception {
 
             //then
             mvc.perform(MockMvcRequestBuilders.delete(BASE_URL + "/" + gameId))
```

Comment:
```suggestion
            mvc.perform(delete(BASE_URL + "/" + gameId))
```
Wszędzie dodałbym statyczne importy tak samo jak masz w klasie RatingControllerTest